### PR TITLE
Create apa-6th-swedish-legislations-university-of-gothenburg.csl

### PR DESCRIPTION
**Pull request by magnuskull**
Style providing support for references to Swedish legislation, motions and propositions as well as to Official Reports of the Swedish Government.

### CODE REVIEW

**Comment by adam3smith** [comment]
Is the style as needed documented somewhere? And there is no way to make this work with the generic APA style simply by adjusting the Swedish locale in there?

**Comment by rmzelle** [comment]
> Is the style as needed documented somewhere?

@adam3smith, in case you hadn't seen it yet, the style has a documentation link to https://github.com/ub-digit/citation-styles/tree/master/zotero/apa-6th-swedish-legislations-university-of-gothenburg, which has some documentation.

@magnuskull, we're still interested in hearing whether there is any more documentation by the University of Gothenburg on these citation requirements.

>And there is no way to make this work with the generic APA style simply by adjusting the Swedish locale in there?

I doubt it. @magnuskull, will this style be used in Swedish and English? It currently has some hard-coded English translations in it.

**Comment by rmzelle** [comment]
(also, we probably want to change the style title a little, e.g. to "University of Gothenburg - Swedish legislations" or the Swedish version of that, but we can decide that when it's clear what language(s) this style will be used in, and once we have seen all documentation)

**Comment by rmzelle** [comment]
@magnuskull, closing because of inactivity. We can always reopen this if you get back to us.

**Comment by bjornekstrom** [comment]
@rmzelle, I'm replying on behalf of @magnuskull.

As for your questions:

* With regards to the test report, these issues have been sorted out.
* The documentation linked above is the one we have. Is anything missing from it?
* Our primary target languages for the stylesheet are English and Swedish, but there's nothing that says that the style can't be used in other languages.
* The hardcoded English translations have been removed. The hardcoded Swedish translation being kept changes the phrasing of "accessed from" to "accessed" ("tillgänglig vid" to "tillgänglig") as per the APA reference style guidelines of University of Gothenburg.
* Please settle on a style title and we'll make changes accordingly.

Please also open this again for us to make new pull requests.

**Comment by rmzelle** [comment]
@bjornekstrom, please go ahead and update the pull request. We can figure out the best title afterwards.

**Comment by bjornekstrom** [comment]
@rmzelle, It seems now like the stylesheet went through the test.

Perhaps "University of Gothenburg - APA (Swedish legislations)" would work as a title?

**Comment by rmzelle** [comment]
@adam3smith, please merge if you're okay with this. (my only remaining (minor) complaint is that some of the XML comments refer to "författning/statute" as an item type, neither of which exist in CSL 1.0.1)

**Comment by bjornekstrom** [comment]
@rmzelle Thanks for your help. I was aware that the link was broken but this has now been fixed on the ub-digit repository. The link has also been directed to the README file.

**Comment by rmzelle** [comment]
(@adam3smith, gentle bump)

**Comment by bjornekstrom** [comment]
@rmzelle The comments regarding "författning/statute" are not crucial to us but rather an indication of tiny changes made to the stylesheet for Zotero use. You may remove them and merge the stylesheet if you wish.

**Comment by rmzelle** [comment]
> The comments regarding "författning/statute" are not crucial to us but rather an indication of tiny changes made to the stylesheet for Zotero use. You may remove them and merge the stylesheet if you wish.

Ah, that's fine then (although we could clarify that that we're talking about a Zotero item type in those XML comments). I was just a little confused because CSL itself doesn't have a "statute" item type (the Zotero "statute" item type maps to "legislation" in CSL, https://aurimasv.github.io/z2csl/typeMap.xml#map-statute).

Let's see if @adam3smith has some time to take a look this weekend, otherwise I'll accept this myself.

File: university-of-gothenburg-apa-swedish-legislations.csl [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+              <text term="from" suffix=" "/>
+              <text variable="URL"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <choose>
+          <if variable="DOI">
+            <text variable="DOI" prefix="doi: "/>
```
Comment (adam3smith) [inline university-of-gothenburg-apa-swedish-legislations.csl]: this seems odd -- you probably also want the https://doi.org/ prefix here, no?

Comment (bjornekstrom) [reply university-of-gothenburg-apa-swedish-legislations.csl]: According to our guidelines "doi:" in lowercase is the way to go here.

Please refer to page 6 of our guidelines (although in Swedish) for reference: http://www.utbildning.gu.se/digitalAssets/1366/1366320_apa-lathunden-2012.pdf

Comment (adam3smith) [reply university-of-gothenburg-apa-swedish-legislations.csl]: OK, but then this is incorrect further up: https://github.com/citation-style-language/styles/pull/2779/files#diff-1717d9d6b09f6c2686dcf86ba0e2c385R132 

If you have any way to affect the citation style, btw. using `doi: ` is a bad idea. It stems from a time when people expected it to become its own internet protocol like https: or ftp: The URL form, `https://doi.org/` is now universally preferred.

Comment (rmzelle) [reply university-of-gothenburg-apa-swedish-legislations.csl]: (see also https://www.zotero.org/support/kb/doi_in_apa)

**Comment by rmzelle** [comment]
Thanks! (I changed the DOI prefix to be consistently "doi: " throughout the style, which I assume is what's desired)

**Comment by bjornekstrom** [comment]
Great. Thanks for your help!

**Pull request merged**